cmd/podman/shared/pod.go

```diff
@@ -134,4 +137,129 @@ func CreatePortBindings(ports []string) ([]ocicni.PortMapping, error) {
 	return portBindings, nil
 }
 
+func GetPodsWithFilters(r *libpod.Runtime, c *cliconfig.PodPsValues) ([]*libpod.Pod, error) {
```

Comment:
Comments on public functions are always appreciated

Reply:
Yeah. As mentioned I am working on the documentation just wanted to get the PR out to validate the code flow.

Reply:
Added docs.